downloads/api/oai/shipcloud_v1_oai3.json

```diff
@@ -4321,39 +4343,46 @@
           },
           "item_info": {
             "type": "array",
-            "description": "Information about this variant version of the product",
-            "uniqueItems": false,
+            "uniqueItems": true,
+            "minItems": 0,
+            "description": "Information about this variant version of the product. Used to display a (localized) label and value pair to describe an item's property, e.g. a product category.",
             "items": {
               "type": "object",
               "properties": {
                 "name": {
-                  "type": "object",
-                  "description": "Name of the product variant",
-                  "properties": {
-                    "de": {
-                      "type": "string",
-                      "description": "German name of the product variant property"
-                    },
-                    "fallback": {
-                      "type": "string",
-                      "description": "Fallback name of the product variant property"
-                    }
-                  }
+                  "$ref": "#/components/schemas/localized_attributes"
                 },
                 "value": {
-                  "type": "object",
-                  "description": "Value of the product variant property",
-                  "properties": {
-                    "de": {
-                      "type": "string",
-                      "description": "Value of the product variant property in German"
-                    },
-                    "fallback": {
-                      "type": "string",
-                      "description": "Fallback value of the product variant property"
-                    }
-                  }
+                  "$ref": "#/components/schemas/localized_attributes"
                 }
+              },
+              "required": [
+                "name",
+                "value"
+              ]
+            }
+          },
+          "gross_price": {
+            "type": "number",
+            "description": "Gross price of an item. Used to calculate the refund amount of a return shipment (a Return Portal Plus feature)."
+          },
+          "returnability": {
+            "type": "object",
+            "description": "Custom message displayed to the customer when an item cannot be selected for a return shipment.",
```

Comment:
I think that this description would be better at the ``nonreturnable_reason`` property. Here, we should roughly describe the effect of the ``returnability`` object as a whole.

Reply:
Problem is, `nonreturnable_reason` cannot have a distinct description, because of the `$ref`